OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -257,6 +260,22 @@ public void showNoNetworkDialog() {
     }
 
     @Override
+    public boolean dispatchTouchEvent(MotionEvent ev) {
+        Animation errorMessageAnim = activityLoginBinding.errorLayout.getAnimation();
+        if (errorMessageAnim == null || errorMessageAnim.hasEnded()) {
+            ViewAnimationUtil.hideMessageBar(activityLoginBinding.errorLayout);
+        }
+        return super.dispatchTouchEvent(ev);
+    }
```

Comment:
You don't need this implementation if you're not using the fly-in messages in this Activity.